src/main/java/hudson/plugins/claim/ClaimPublisher.java

```diff
@@ -63,11 +74,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         }
 
-        return true;
     }
 
     public BuildStepMonitor getRequiredMonitorService() {
-        return BuildStepMonitor.STEP;
+        return BuildStepMonitor.NONE;
     }
```

Comment:
Following the suggestions at http://javadoc.jenkins-ci.org/jenkins/tasks/SimpleBuildStep.html